src/Firebase.cpp

```diff
@@ -118,11 +118,12 @@ FirebaseCall::FirebaseCall(const String& host, const String& auth,
   }
 }
 
-const JsonObject& FirebaseCall::json() {
-  //TODO(edcoyne): This is not efficient, we should do something smarter with
+JsonObject& FirebaseCall::parseJson() {
+  // TODO(edcoyne): This is not efficient, we should do something smarter with
   //the buffers.
   buffer_ = DynamicJsonBuffer();
-  return buffer_.parseObject(response());
+  // NOTE(proppy): this effectively void the response_ buffer.
+  return buffer_.parseObject(const_cast<char*>(response_.c_str()));
```

Comment:
While efficient I think we should just make a mutable copy of the response for now to pass in. This seems super "premature optimization" hack.